scaling_performance/optimizing_compute_resources.adoc

```diff
@@ -92,12 +92,13 @@ Registry  credentials.
 ====
 
 [[scaling-performance-debugging]]
-== Debugging {product-title} Using the RHEL Tools Container
+== Debugging Using the RHEL Tools Container Image
 
-Red Hat distributes a *rhel-tools* container, which:
+Red Hat distributes a *rhel-tools* container image, packaging tools that aid in
+debugging scaling or performance problems. This container image:
 
-* Allow users to deploy minimal footprint container hosts by moving packages out of the base distribution and into this support container.
-* Provide debugging capabilities for Red Hat Enterprise Linux 7 Atomic Host, which has an immutable packet tree. *rhel-tools* includes utilities such as tcpdump, sosreport, git, gdb, perf, and many more common system administration utilities.
+* Allows users to deploy minimal footprint container hosts by moving packages out of the base distribution and into this support container.
+* Provides debugging capabilities for Red Hat Enterprise Linux 7 Atomic Host, which has an immutable packet tree. *rhel-tools* includes utilities such as tcpdump, sosreport, git, gdb, perf, and many more common system administration utilities.
```

Comment:
`s/packet/package/` here; this supplies tools that can't be installed as packages on AH. Not sure what an immutable packet tree would mean :)